cmd_vel_params:
    
    default_x_linear: "/teleop/swerve_drive_controller/cmd_vel"
    default_y_linear: "/teleop/swerve_drive_controller/cmd_vel"
    default_z_angular: "/teleop/swerve_drive_controller/cmd_vel"
    
    pub_delay: 0.01 #limits publishing to 100 Hz

    subscriber_topics: [ "/align_hatch/align_hatch/swerve_drive_controller/cmd_vel", "/climber/swerve_drive_controller/cmd_vel", "/hatch_outtake/swerve_drive_controller/cmd_vel", "/path_follower/path_follower_pid/swerve_drive_controller/cmd_vel", "/teleop/swerve_drive_controller/cmd_vel" ]
